examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>I think "Applying to your own dataset" has a higher motivation factor for users to read the section than "Integratd pipeline with"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>See my comment to the previous message</p>